src/pages/404.js

```diff
@@ -12,4 +15,11 @@ const Page404 = () => (
   </div>
 )
 
-export default Page404
+export default withUnpublishedPreview(Page404, {
+  templateMap: {
+    page: PageTemplate,
+    homepage: HomepageTemplate,
+    prismicPage: PageTemplate,
+    prismicHomepage: HomepageTemplate,
```

Comment:
Why do you need to have `page` & `prismicPage`? Likewise for the 'homepage'?

Reply:
Here's the docs on `withUnpublishedPreview`
I was seeing this error a lot https://github.com/angeloashmore/gatsby-source-prismic/blob/master/src/withUnpublishedPreview.tsx#L29-L33 I can look into it :)

Reply:
Apparently it needs both: https://github.com/angeloashmore/gatsby-source-prismic/issues/246